css/style.css

```diff
@@ -1,260 +1,241 @@
 .tabsAside.background
 {
-	z-index: 9999 !important;
-	background-color: rgba(255, 255, 255, .5) !important;
-	position: fixed !important;
-	top: 0 !important;
-	bottom: 0 !important;
-	right: 0 !important;
-	left: 0 !important;
-	transition: .2s !important;
-	opacity: 0;
-	color: black !important;
+	z-index: 9999;
```

Comment:
This trick should not be needed anymore : Since we are in the iframe, we inherit the insane z-index of the iframe already.
```suggestion
```